pixart.js

```diff
@@ -1 +1,13 @@
-
+$(document).ready(function(){
+  $("#set-color").on("click", function(e){
+    e.preventDefault();
+    console.log("test!");
```

Comment:
I'd remove `console.log` statements once you're done debugging. Want to keep that code clean and readable!